Assets/Scripts/SS3D/Systems/Inventory/Items/Identification/PDA.cs

```diff
@@ -15,18 +15,18 @@ namespace SS3D.Systems.Inventory.Items.Generic
     public class PDA : Item, IIdentification
```

Comment:
The filters were removed from the PDA, added them back

Reply:
done